### DISCUSSION TITLE
MissingRate at /customer Rate GBP-> USD does not exist

### DISCUSSION THREAD

**Discussion by EdaDeniz** [django-money/django-money#609] [Q&A] [answered]
I get an error:

MissingRate at /customer Rate GBP -> USD does not exist

How can I solve it?

Detailed question is in here: https://stackoverflow.com/questions/66905372/how-to-solve-missingrate-error-in-django

**Comment by Stranger6667** [comment]
Hi! Did you run `python manage.py update_rates` ?

**Comment by EdaDeniz** [comment]
Yes I run it but did not work

Dmitry Dygalo ***@***.***>, 1 Nis 2021 Per, 16:35 tarihinde
şunu yazdı:

> Hi! Did you run python manage.py update_rates ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/django-money/django-money/discussions/609#discussioncomment-557613>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKBY3X64Q6YY57WYZ2FJOYDTGRZAJANCNFSM42HDQB7Q>
> .
>


-- 
Eda Deniz ÇELİK
Computer Engineering

**Reply by Stranger6667** [reply]
What was the output then? I'd expect that `djmoney` raises `MissingRate` if the `update_rates` command fails

**Comment by EdaDeniz** [comment, marked as answer]
Hi Mr. Dygalo,
Firstly thanks a lot for your e-mail it was a great kindness. And sorry for
bothering you again.
I solved the problem.
In my settings there is a line:

OPEN_EXCHANGE_RATES_URL =
'https://openexchangerates.org/api/historical/2017-01-01.json?symbols=EUR,NOK,SEK,CZK,USD'

I should add every currency that I want to use at end of this link. I added
all currencies then made migrate and finally, I updated the rates
./manage.py
 update rates. The program started running. My problem solved.

Thanks for your help.

Dmitry Dygalo ***@***.***>, 1 Nis 2021 Per, 16:48 tarihinde
şunu yazdı:

> What was the output then? I'd expect that djmoney raises MissingRate if
> the update_rates command fails
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/django-money/django-money/discussions/609#discussioncomment-557677>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKBY3X5QRJWFEKMJ5JYBA7TTGR2RJANCNFSM42HDQB7Q>
> .
>


-- 
Eda Deniz ÇELİK
Computer Engineering
